src/Common/tests/System/PlatformDetection.cs

```diff
@@ -43,6 +43,33 @@ public static partial class PlatformDetection
 
         public static int WindowsVersion => GetWindowsVersion();
 
+        public static bool IsNetfx462OrNewer()
+        {
+            if (!IsFullFramework)
+            {
+                return false;
+            }
+
+            string[] descriptionArray = RuntimeInformation.FrameworkDescription.Split(' ');
```

Comment:
Why not break out `GetFrameworkVersion()` returning a Version. That way someone can easily write IsNetfx470OrNewer() etc later.